ethcore/src/block.rs

```diff
@@ -339,22 +337,6 @@ impl<'x> OpenBlock<'x> {
 		Ok(())
 	}
 
-	/// Populate self from a header.
```

Comment:
This block should be reverted. If we don't want to set `extraData` for clique here, then maybe a way to do it would be to add `Engine::allow_custom_extra_data() -> bool` and condition on that.

Reply:
No, this block is actually used only once, and it wasn't very clear what is function is for, right now I just let caller of OpenBlock::new() to figure out what header they want to populate.